target/zephyr/Kconfig

```diff
@@ -149,6 +149,82 @@ if MENDER_MCU_CLIENT
                 the bootloader configuration, e.g. `sysbuild.conf` if you're using sysbuild.
     endmenu
 
+    menu "Mender Artifact configuration"
+
+        config MENDER_ARTIFACT_GENERATE
+            bool "Generate a Mender Artifact from the built Zephy kernel image"
+            default y
+            help
+                Auto-generates a Mender Artiafact from the Zephyr kernel. It requires the mender-artifact CLI tool
+                installed in the workstation.
+
+        if MENDER_ARTIFACT_GENERATE
+
+            config MENDER_ARTIFACT_NAME
```

Comment:
It would be nice (thinking of myself as a Mender MCU developer) if this one could have a magic name. E.g., `artifact_XXXXXX`, that substituted the X'es with an increasing sequence number, timestamp or something similar. Kind of like the `mkstemp(3)` function. This way you could quickly generate a series of artifacts without having to change the _Kconfig_ each time.

This is ofc out of scope for this PR. Maybe a future value add project idea for me :wink: Let me know what you think!

Reply:
Thank you for the suggestion. We will discuss this topic in today's Meeting and I'll bring up your suggestion.

Reply:
Sounds like a really nice feature to me.